clients/privacy-center/public/fides-js-demo.html

```diff
@@ -177,6 +177,30 @@ <h2>Property id</h2>
       } else {
         window.addEventListener("FidesInitialized", onInitialized);
       }
+
+      window.addEventListener("FidesUIShown", () => {
+        // Log event timing
+        const fidesLoaded = performance
+          .getEntriesByType("resource")
```

Comment:
Can performance.getEntriesByType("resource") ever be null?

Reply:
not null, only empty []